### PR TITLE
Wip doc landing page update

### PR DESCRIPTION
**Pull request by zdover23**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by batrick** [comment]
jenkins render docs

File: doc/index.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ Installing the First Node
 
 #. Using the ip address that you discovered in the step immediately prior to this step, run the following command::
 
-   $ sudo ./cephadm bootstrap --mon-ip 192.168.1.101 --output-config ceph.conf --output-keyring ceph.keyring --output-pub-ssh-key ceph.pub 
+   $ sudo ./cephadm bootstrap --mon-ip 192.168.1.101 --output-pub-ssh-key ceph.pub
```
Comment (liewegas) [inline doc/index.rst]: we can leave off the output pub ssh key option too-- that also defaults to 'ceph.pub'.

**Comment by liewegas** [comment]
I wonder if we should have just the bootstrap stage on the front page and then link to the longer bootstrap.rst, which walks through deploying additional daemons?

File: doc/index.rst [hunk 2]
```diff
@@ -17,9 +17,9 @@ Installing the First Node
 -------------------------
 
 1. Install a recent, supported Linux distribution on a computer.
-2. Install docker. On Fedora or Centos::
-   
-   $ sudo dnf install docker 
+2. Install podman or docker. On Fedora or Centos::
+
+   $ sudo dnf install podman
```
Comment (sebastian-philipp) [inline doc/index.rst]: Hm. you guys are still sure you want to duplicate doc/cephadm/index.rst in doc/index.rst? Or should these changes go into cephadm/index.rst?

Comment (zdover23) [reply doc/index.rst]: I vote for Sebastian's plan.

**Pull request closed without merging**